### DISCUSSION TITLE
关于wangEditor-next的editor-for-vue2，for-vue3和for-react

### DISCUSSION THREAD

**Discussion by ruixingshi** [wangeditor-next/wangEditor-next#145] [Q&A] [answered]
作者你好，非常高兴看到这个优秀的开源项目得以有人继续维护，我有一些问题想讨论一下：
Q1：上述三个库后续也有由作者迁移过来维护吗？我看现在只是fork了
Q2：上述三个库为什么不作为wangEditor monorepo的一部分，而是拆库单独发包？

**Comment by cycleccc** [comment, marked as answer]
A1：不只是 fork，还有其它的修复更新。（后续变动大了也会取消 fork 单独维护）
A2：这个属于历史问题，原来这三个库也是 monorepo 的一部分，后来双越老师又拆出来了，具体可以看 https://github.com/cycleccc/wangEditor-next/commit/cf03f08eed5799c4fa64713604a6c4b85e73222e
但是没有详细的说明为什么拆包

**Reply by ruixingshi** [reply]
感谢回复，关于Q2，后续你这边有打算合并回来吗？是否可以看下或者问下双越老师呢。个人觉得在一起更符合直觉。

**Reply by cycleccc** [reply]
双越老师不太支持合并在一起，具体为什么不这么做，可能得试试才能知道。
有这个打算，但优先级不算很高，有兴趣推动这个事吗？

**Reply by cycleccc** [reply]
可以参考 tiptap ，完善各个基础插件和平台包的管理,现在在处理微前端的一个 bug，这周或下周着手完善吧。
> https://github.com/ueberdosis/tiptap/tree/develop/packages/vue-3

**Reply by ruixingshi** [reply]
> 双越老师不太支持合并在一起，具体为什么不这么做，可能得试试才能知道。 有这个打算，但优先级不算很高，有兴趣推动这个事吗？

后续有机会我可以试试。
